tests/utils.py

```diff
@@ -21,9 +21,33 @@ def spec_from_config_file(filepath: str) -> cubed.Spec:
     return spec_from_config(config)
 
 
+def get_directory_size(work_dir: str) -> float:
+    """Get the size of any data written to the given directory (local or remote)."""
+
+    import fsspec
+
+    total_size = 0
+
+    if work_dir.startswith('s3://'):
+        fs = fsspec.filesystem('s3')
```

Comment:
A simpler way to do this (that I only discovered [yesterday](https://github.com/cubed-dev/cubed-benchmarks/blob/main/tests/benchmarks/test_array.py#L85)!) is

```python
fs, _, _ = fsspec.get_fs_token_paths(work_dir)
```